### PR TITLE
Update GiulioSorrentino.wxBriscola.installer.yaml

### PR DESCRIPTION
**Pull request by GiulianoSpaghetti**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/167157)

### CODE REVIEW

**Comment by GiulianoSpaghetti** [comment]
Attention: it's a c++ excutable in wxwidgets for both amd64 and arm64, but it requirers for both version the vcredist x64.
I don't know how to write vcredist arm64 if platofrmo is arm64

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Regarding your comment above, you can include dependencies at the installer level as:

```
# Created using wingetcreate 1.6.1.0
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json

PackageIdentifier: GiulioSorrentino.wxBriscola
PackageVersion: 0.6.3
InstallerType: wix
ProductCode: '{76833DA3-34C1-49B0-A918-8A1825DD50FE}'
Installers:
- Architecture: x64
  InstallerUrl: https://github.com/numerunix/wxBriscola-for-Windows/releases/download/0.6.3/wxBriscola-0.6.3-1-amd64.msi
  InstallerSha256: 57286C65E1870A1B6280808EEC7694E03084B86598E55152783786EF5A7504A2
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
- Architecture: arm64
  InstallerUrl: https://github.com/numerunix/wxBriscola-for-Windows/releases/download/0.6.3/wxBriscola-0.6.3-1-arm64.msi
  InstallerSha256: F2013BBFEB95C9EBC6B6DAF42F90E6319F43184BB7863B3931CF72E34D81DB4C
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2019.arm64
ManifestType: installer
ManifestVersion: 1.6.0

```

File: manifests/g/GiulioSorrentino/wxBriscola/0.6.3/GiulioSorrentino.wxBriscola.installer.yaml [hunk 1]
```diff
@@ -4,13 +4,19 @@
...
+  InstallerSha256: 57286C65E1870A1B6280808EEC7694E03084B86598E55152783786EF5A7504A2
+    Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+- Architecture: arm64
+  InstallerUrl: https://github.com/numerunix/wxBriscola-for-Windows/releases/download/0.6.3/wxBriscola-0.6.3-1-arm64.msi
+  InstallerSha256: F2013BBFEB95C9EBC6B6DAF42F90E6319F43184BB7863B3931CF72E34D81DB4C
+    Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2022.arm64
```
Comment (mdanish-kh) [inline manifests/g/GiulioSorrentino/wxBriscola/0.6.3/GiulioSorrentino.wxBriscola.installer.yaml:20]:
```suggestion
  Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
- Architecture: arm64
  InstallerUrl: https://github.com/numerunix/wxBriscola-for-Windows/releases/download/0.6.3/wxBriscola-0.6.3-1-arm64.msi
  InstallerSha256: F2013BBFEB95C9EBC6B6DAF42F90E6319F43184BB7863B3931CF72E34D81DB4C
  Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2022.arm64
```

Indentation needs to be fixed

**Comment by mdanish-kh** [comment]
@GiulianoSpaghetti

https://github.com/microsoft/winget-pkgs/pull/167157#discussion_r1712957660 isn't resolved. The indentation for `Dependencies` node is still incorrect, which is why you're seeing https://github.com/microsoft/winget-pkgs/labels/Manifest-Validation-Error

**Comment by GiulianoSpaghetti** [comment]
Thanks

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**